modules/nf-core/platypus/main.nf

```diff
@@ -0,0 +1,55 @@
+/*
+unfortunately need to output the version manually
+becouse platypus CallVariants does not include --version or -v commend
+*/
+process PLATYPUS {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::platypus-variant=0.8.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/platypus-variant:0.8.1--py27_1':
+        'quay.io/biocontainers/platypus-variant:0.8.1--py27_1' }"
+
+    input:
+
+    tuple val(meta), path(tumor_file), path(tumor_file_bai), path(control_file),  path(control_file_bai)
+    path fasta
+    path fai
+    path skipregions_file
+
+    output:
+    tuple val(meta), path('*.vcf.gz')            , emit: vcf
+    tuple val(meta), path('*.vcf.gz.tbi')        , emit: vcf_tbi
```

Comment:
```suggestion
    tuple val(meta), path('*.vcf.gz.tbi')        , emit: tbi
```

most other modules just call it `tbi` since it's shorter to type :)

Reply:
applied